pom.xml

```diff
@@ -521,7 +540,8 @@
                 <artifactId>maven-checkstyle-plugin</artifactId>
                 <version>${maven-checkstyle-plugin.version}</version>
                 <configuration>
-                    <configLocation>src/main/resources/dd_checks.xml</configLocation>
+                    <configLocation>src/main/resources/checkstyle_ci.xml</configLocation>
```

Comment:
For report should use `checkstyle.xml`, not `checkstyle_ci.xml`.
This pr add 2 template files of checkstyle, is it?

Reply:
Yes, we add 2 temlate files of checkstyle. I thought checkstyle.xml was for the IDE. I will change the setting.